src/main/java/in/projecteka/consentmanager/clients/DataRequestNotifier.java

```diff
@@ -9,12 +11,15 @@
 import java.util.function.Supplier;
 
 import static in.projecteka.consentmanager.clients.ClientError.unknownErrorOccurred;
+import static in.projecteka.consentmanager.common.Constants.HDR_HIP_ID;
 import static org.springframework.http.HttpHeaders.AUTHORIZATION;
 
 @AllArgsConstructor
 public class DataRequestNotifier {
     private final WebClient.Builder webClientBuilder;
     private final Supplier<Mono<String>> tokenGenerator;
+    private final GatewayServiceProperties gatewayServiceProperties;
+    private final String GATEWAY_DATAFLOW_URL = "%s/health-information/hip/request";
```

Comment:
you can get GATEWAY_DATAFLOW_URL from constants

Reply:
done